main/res/layout/cache_filter_generic_string.xml

```diff
@@ -54,8 +54,6 @@
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/searchtext"
         android:hint="@string/cache_filter_stringfilter_searchtext_hint"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
         style="@style/textinput_dropdown"
         app:endIconMode="clear_text"
```

Comment:
Is it possible to add "app:endIconMode="clear_text" also to the "textinput_*" style?

Reply:
I have tried that, but the compiler complained about not knowin "app:endIconMode" attribute in `styles.xml`, so I keept it in the layout files.

Reply:
I googled this quickly and found this: https://stackoverflow.com/questions/39336054/using-app-namespace-in-style
It seems that for "app" attributes in styles, you simply leave out the "app:" praefix

Reply:
Yeah, should have known that. Have used that many times in the past, but it did not come into my mind yesterday... :-(

Adjusted the styles accordingly.